README.rst

```diff
@@ -586,6 +586,16 @@ by disabling both web and email option for a notification setting.
 then ``email_notification`` or ``web_notification`` option of concerned notification type will be used
 respectively.
 
+Object Notification Preferences
+-------------------------------
+*OpenWISP Notifications* allows users to even define notification preferences based on target object.
+Users can disable notifications for any object for a desired period of time.
+Using the widget on an object's admin form, a user can disable notifications for that object
+for a day, week, month or even forever.
+
+**Note**: This feature requires configuring `"OPENWISP_NOTIFICATIONS_IGNORE_ENABLED_ADMIN" <#openwisp-notifications-object-notification-widget>`_
```

Comment:
the internal reference seems mistaken, should be `openwisp_notifications_ignore_enabled_admin`